src/modals/report-user.js

```diff
@@ -58,7 +58,7 @@ async function reportUserHandler(interaction) {
 
 	const reportEmbed = new Discord.EmbedBuilder();
 
-	reportEmbed.setColor(0xC0C0C0);
+	reportEmbed.setColor(0xF36F8A);
```

Comment:
Why was this color changed?

Reply:
This color was changed to be consistent with the other colors changed.